css/signuppages.css

```diff
@@ -0,0 +1,314 @@
+      color: #999; }
+    .form-input::-webkit-input-placeholder {
+      font-weight: 500; }
+    .form-input::-moz-placeholder {
+      font-weight: 500; }
+    .form-input:-ms-input-placeholder {
+      font-weight: 500; }
+    .form-input:-moz-placeholder {
+      font-weight: 500; }
+    .form-input:focus {
+      border: 1px solid transparent;
+      -webkit-border-image-source: -webkit-linear-gradient(to right, #9face6, #74ebd5);
+      -moz-border-image-source: -moz-linear-gradient(to right, #9face6, #74ebd5);
+      -o-border-image-source: -o-linear-gradient(to right, #9face6, #74ebd5);
+      border-image-source: linear-gradient(to right, #9face6, #74ebd5);
+      -webkit-border-image-slice: 1;
+      border-image-slice: 1;
+      border-radius: 5px;
+      -moz-border-radius: 5px;
+      -webkit-border-radius: 5px;
+      -o-border-radius: 5px;
+      -ms-border-radius: 5px;
+      background-origin: border-box;
+      background-clip: content-box, border-box; }
+      .form-input:focus::-webkit-input-placeholder {
+        color: #222; }
+      .form-input:focus::-moz-placeholder {
+        color: #222; }
+      .form-input:focus:-ms-input-placeholder {
+        color: #222; }
+      .form-input:focus:-moz-placeholder {
+        color: #222; }
+  
+  .form-submit {
+    width: 100%;
+    border-radius: 5px;
+    -moz-border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -o-border-radius: 5px;
+    -ms-border-radius: 5px;
+    padding: 17px 20px;
+    box-sizing: border-box;
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+    text-transform: uppercase;
+    border: none;
+    background-image: -moz-linear-gradient(to left, #61d4bf, #26752b);
+    background-image: -ms-linear-gradient(to left, #61d4bf, #26752b);
+    background-image: -o-linear-gradient(to left, #61d4bf, #26752b);
+    background-image: -webkit-linear-gradient(to left, #61d4bf, #26752b);
+    background-image: linear-gradient(to left, #61d4bf, #26752b); }
+  
+  input[type=checkbox]:not(old) {
+    width: 2em;
+    margin: 0;
+    padding: 0;
+    font-size: 1em;
+    display: none; }
+  
+  input[type=checkbox]:not(old) + label {
+    display: inline-block;
+    margin-top: 7px;
+    margin-bottom: 25px; }
+  
+  input[type=checkbox]:not(old) + label > span {
+    display: inline-block;
+    width: 13px;
+    height: 13px;
+    margin-right: 15px;
+    margin-bottom: 3px;
+    border: 1px solid #ebebeb;
+    border-radius: 2px;
+    -moz-border-radius: 2px;
+    -webkit-border-radius: 2px;
+    -o-border-radius: 2px;
+    -ms-border-radius: 2px;
+    background: white;
+    background-image: -moz-linear-gradient(white, white);
+    background-image: -ms-linear-gradient(white, white);
+    background-image: -o-linear-gradient(white, white);
+    background-image: -webkit-linear-gradient(white, white);
+    background-image: linear-gradient(white, white);
+    vertical-align: bottom; }
+  
+  input[type=checkbox]:not(old):checked + label > span {
```

Comment:
Codacy found an issue: [Missing vendor-prefixed CSS gradients for Old Webkit (Safari 4+, Chrome).](https://app.codacy.com/gh/BuildForSDGCohort2/team-040-frontend/pullRequest?prid=6093393)